notebooks/MAST/find_eclipse_candidates/find_eclipse_candidates.ipynb

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
<p>Correct spelling of "companions" in the second sentence</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>